lib/Graph/CMakeLists.txt

```diff
@@ -17,6 +17,19 @@ add_custom_target(AutoGenNode
                      "${NODES_DEF}")
 add_dependencies(AutoGen AutoGenNode)
 
+find_package(Git)
+# Get the commit's SHA1, author name, date and subject
+execute_process(COMMAND
```

Comment:
side note: make sure to incorporate this in fbcode with buck TARGETS before merging.

Reply:
checked with @bertmaher about embedding build info/commit hashes in our fbcode binary, and it seems that it's already taken care of, and can be invoked with certain buck flags.